logger/logger.go

```diff
@@ -36,6 +37,9 @@ func ResetLogs() {
 
 // Log - handles adding logs
 func Log(verbosity int, message ...string) {
+	var mu sync.Mutex
```

Comment:
What is the mutex for?

Reply:
The logger is being accessed by 3 different go routines in v0.10.0 and this helps avoid concurrent map writes. Tested and it doesn't affect performance

Reply:
has been removed